### PR TITLE
AT-49 Mobile Menu, AT-45 Store Most Searched Artists

### PR DESCRIPTION
**Pull request by berabulut**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines. For the guidelines check the README.
- [ ] Tests for the changes have been added (for bug fixes / features). As we discussed this is optional but encouraged.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR introduces a menu for mobile devices, feature that lets us store searched artists, count them and fetching the information when we need it.

* **What is the current behavior?** (You can also link to an open issue here)
Currently no mobile menu and db action.

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

You need to create .env file that includes following lines:

DB_HOST
DB_USER
DB_PASS
DB_NAME=AfternoonTeam

### CODE REVIEW

File: backend/.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DB_HOST=database-2.cgsskta75qvp.us-west-2.rds.amazonaws.com
```
Comment (Miguel-Villanueva) [inline backend/.env]: is this a security risk

File: backend/index.js [hunk 2]
```diff
@@ -12,11 +12,26 @@ var request = require('request'); // "Request" library
 var cors = require('cors');
 var querystring = require('querystring');
 var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser')
 
-var client_id = ''; // Your client id
-var client_secret = ''; // Your secret
+var client_id = 'c0de84ab300244c5b319bda9bcacb641'; // Your client id
```
Comment (Codubee) [inline backend/index.js]: This one as well

Comment (Codubee) [reply backend/index.js]: Have to remove and create new keys

**Comment by berabulut** [comment]
My merged PR was containing this commits so i closed this PR.

**Pull request closed without merging**